src/tpm2-tss-engine-common.h

```diff
@@ -33,6 +33,8 @@
 #ifndef TPM2_TSS_ENGINE_COMMON_H
 #define TPM2_TSS_ENGINE_COMMON_H
 
+#include <stdatomic.h>
```

Comment:
I wonder how portable this C11 construct is, any ideas?

Reply:
With the detection of C11 atomics and then fallback to gcc/clang builtins then this should cover most cases except for MSVC on Windows. If that needs to be supported then the atomic_fetch_add could be implemented in terms of the native InterlockedExchangeAdd API, or the OpenSSL CRYPTO_atomic_add API.

Reply:
I didn't even realize openessl had an atomic API. It seems that's since OpenSSL_1_1_1, but I could be wrong. Right now configure shows the minimum to be version 1.0.2g, so I'd like to not bump us out of 1.0.2 land, but I could be persuaded.

I like just using OSSL and then we don't have to worry about C11 or compiler support. I don't like that we have to bump the OSSL version

@AndreasFuchsSIT any thoughts here?